app/src/controllers/testCase/selectors.ts

```
@@ -78,6 +82,8 @@ export const testCasesPageSelector = (state: RootState): Page | null =>
 
 export const testCaseDetailsSelector = (state: RootState) => state.testCase?.details?.data;
 
+export const testCaseDetailsSelector = (state: RootState) => state.testCase?.details?.data;
```

Biome (2.3.13)

[error] Shouldn't redeclare 'testCaseDetailsSelector'. Consider to delete it or rename it.

'testCaseDetailsSelector' is defined here:


(lint/suspicious/noRedeclare)